### PR TITLE
Update performance.rst

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Promoting the term "Stopwatch" to improve Google findability, and also https://symfony.com/doc/4.4/index.html
See https://github.com/symfony/symfony-docs/issues/14476

### CODE REVIEW

File: performance.rst [hunk 1]
```diff
@@ -211,8 +211,8 @@ deployment process too):
 
 .. _profiling-applications:
 
-Profiling Applications
-----------------------
+Profiling Applications and Stopwatch
```
Comment (Matts) [inline performance.rst:214]: Hi @ThomasLandauer 

Shouldn't this be the following?

```suggestion
Profiling Applications with Stopwatch
```

Comment (ThomasLandauer) [reply performance.rst:214]: I'm using the Stopwatch to measure how long a Console Command is taking to run. That's it. I wouldn't call that "profiling" - would you? So the message I'm trying to get across with the word "and" is: "It is a stopwatch; you can use it for profiling; but you can also just use it as a, well, stopwatch."
But I must admit that this message certainly doesn't get across fully ;-)

**Comment by javiereguiluz** [comment]
Thomas, I thank your for proposing this but I'm going to close it as "won't merge". The reason is that this page already includes the "stopwatch" word tens of times. Here are some of those occurrences:

image

Thanks for understanding!

**Comment by ThomasLandauer** [comment]
I have 3 counter arguments:
* When searching for "symfony stopwatch" Google is still showing https://symfony.com/doc/3.2/components/stopwatch.html as top result, and the current page not at all. This is probably due to "stopwatch" not being a heading.
* When searching for "stopwatch" at Symfony's internal search at https://symfony.com/doc/current/index.html there's no "stopwatch" header (only fine print) which is a shame.
* What harm is it doing to include the very component's name in the header?

**Comment by ThomasLandauer** [comment]
ping @javiereguiluz :-)

**Comment by javiereguiluz** [comment]
See #14502.

**Pull request closed without merging**